services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/services/service_tracker/_models.py

```diff
@@ -115,9 +132,28 @@ def set_last_status_notification_to_now(self) -> None:
     ### SERIALIZATION ###
     #####################
 
+    def to_dict(self) -> dict:
+        data = asdict(self)
+        if self.project_id:
+            data["project_id"] = f"{self.project_id}"
+        if self.dynamic_service_start:
+            data["dynamic_service_start"] = self.dynamic_service_start.json()
+        return data
+
+    @classmethod
+    def from_dict(cls, data: dict) -> "TrackedServiceModel":
+        if project_id := data.get("project_id"):
+            data["project_id"] = ProjectID(project_id)
+        if dynamic_service_start := data.get("dynamic_service_start"):
+            data["dynamic_service_start"] = DynamicServiceStart.parse_raw(
+                dynamic_service_start
+            )
+        return cls(**data)
+
```

Comment:
same here, why do you need these specials? and why not using pydantic if you need parsing? parse_obj_as() would do the trick no?

Reply:
replaced with pedantic model.